### PR TITLE
8283387: [macos] a11y : Screen magnifier does not show selected Tab 

### PR DESCRIPTION
**Pull request by azuev-java**
It is impossible right now to manipulate state of the accessible children of the JTabbedPane via their AccessibleValue and because of that some of the assistive technologies are unable to interact with the parts of the JTabbedPane - namely, buttons, representing the JTabbedPane panel navigation buttons. Magnified is just one of the examples that can be engaging without connecting the hardware such as eye movement tracker.

By adding AccessibleValue to the JTabbedPane children containers we eliminate is problem. And since this inner class is private the standard review should be sufficient, no CSR is required.

---------
### Progress
- [x] Change must not contain extraneous whitespace
- [x] Commit message must refer to an issue
- [x] Change must be properly reviewed

### Issue
 * [JDK-8283387](https://bugs.openjdk.java.net/browse/JDK-8283387): [macos] a11y : Screen magnifier does not show selected Tab

### Reviewers
 * [Phil Race](https://openjdk.java.net/census#prr) (@prrace - **Reviewer**)
 * [Alexey Ivanov](https://openjdk.java.net/census#aivanov) (@aivanov-jdk - **Reviewer**)

### Reviewing
<details><summary>Using <code>git</code></summary>

Checkout this PR locally: \
`$ git fetch https://git.openjdk.java.net/jdk pull/8049/head:pull/8049` \
`$ git checkout pull/8049`

Update a local copy of the PR: \
`$ git checkout pull/8049` \
`$ git pull https://git.openjdk.java.net/jdk pull/8049/head`

</details>
<details><summary>Using Skara CLI tools</summary>

Checkout this PR locally: \
`$ git pr checkout 8049`

View PR using the GUI difftool: \
`$ git pr show -t 8049`

</details>
<details><summary>Using diff file</summary>

Download this PR as a diff file: \
<a href="https://git.openjdk.java.net/jdk/pull/8049.diff">https://git.openjdk.java.net/jdk/pull/8049.diff</a>

</details>

### CODE REVIEW

**Review by prrace — APPROVED** [review]

File: test/jdk/javax/accessibility/JTabbedPane/AccessibleTabbedPaneTest.java [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+ * @bug 8283387
+ * @summary [macos] a11y : Screen magnifier does not show selected Tab
+ *          JTabbedPane accessible children had no AccessibleValue manipulation API
+ *          Testing this API since it was added in the fix for 8283387
+ * @run main AccessibleTabbedPaneTest
+ */
+
+public class AccessibleTabbedPaneTest {
+    public static void main(String[] args) {
+        JTabbedPane pane = new JTabbedPane();
```
Comment (aivanov-jdk) [inline test/jdk/javax/accessibility/JTabbedPane/AccessibleTabbedPaneTest.java:39]: Shouldn't this be run on EDT?

Comment (azuev-java) [reply test/jdk/javax/accessibility/JTabbedPane/AccessibleTabbedPaneTest.java:39]: I do not see why. I am only working with the data model and only set the data model state trough the appropriate calls, they do not rely on any asynchronous processing or external events.

Comment (aivanov-jdk) [reply test/jdk/javax/accessibility/JTabbedPane/AccessibleTabbedPaneTest.java:39]: Sounds reasonable.

**Review by aivanov-jdk — APPROVED** [review]

**Pull request closed without merging**